e2e/accessibility_test/accessibility.spec.js

```diff
@@ -0,0 +1,190 @@
+        });
+        it('has a Disabled Continue Main Button', async () => {
+            await expect(screen.carbEntry.DisabledContinueMainButton()).toExist();
+        });
+        it('has a Disabled Continue Header Button', async () => {
+            await expect(screen.carbEntry.DisabledContinueHeaderButton()).toExist();
+        });
+        it('has a Absorption Time Message', async () => {
+            await expect(screen.carbEntry.AbsorptionTimeMessage()).toExist();
+        });
+        it('has a Absorption Time Label', async () => {
+            await expect(screen.carbEntry.AbsorptionTimeLabel()).toExist();
+        });
+        it('has a Date Label', async () => {
+            await expect(screen.carbEntry.DateLabel()).toExist();
+        });
+        it('has a Amount Consumed Label', async () => {
+            await expect(screen.carbEntry.AmountConsumedLabel()).toExist();
+        });
+        it('has a Food Type Label', async () => {
+            await expect(screen.carbEntry.FoodTypeLabel()).toExist();
+        });
+    });
+    describe.skip('override', () => {
+        beforeAll(async () => {
+            await screen.overrides.Open();
+        });
+        afterAll(async () => {
+            await screen.overrides.Cancel();
+        });
+        it('has a Custom Preset Header', async () => {
+            await expect(screen.overrides.CustomPresetHeader()).toExist();
+        });
+        it('has a Cancel Header Button', async () => {
+            await expect(screen.overrides.CancelHeaderButton()).toExist();
+        });
+        it('has a Edit Header Button', async () => {
+            await expect(screen.overrides.EditHeaderButton()).toExist();
+        });
+        it('has a Add Header Button', async () => {
+            await expect(screen.overrides.AddHeaderButton()).toExist();
+        });
+        it('has a Add Preset Message', async () => {
+            await expect(screen.overrides.AddPresetMessage()).toExist();
+        });
+    });
+    describe('settings', () => {
+        beforeAll(async () => {
+            await screen.settings.Open();
+        });
+        afterAll(async () => {
+            await screen.settings.Close();
+        });
+        it('has a done button', async () => {
+            await expect(screen.settings.DoneButton()).toExist();
+        });
+        it('has a configuration header', async () => {
+            await expect(screen.settings.ConfigurationHeader()).toExist();
+        });
+        it('has a pump header', async () => {
+            await expect(screen.settings.PumpHeader()).toExist();
+        });
+        it('has a pump header', async () => {
+            await expect(screen.settings.ContinuousGlucoseMonitorHeader()).toExist();
+        });
+        it('has a settings header', async () => {
+            await expect(screen.settings.SettingsHeader()).toExist();
+        });
+        it('has a Issue Report Label', async () => {
+            //TODO: this is acting like a button!
```

Comment:
[nit] Just pointing out a bunch of TODOs here.

Reply:
yeah - these will become jira cards to update the loop app